packages/maskbook/src/_locales/ja/messages.json

```diff
@@ -358,6 +358,8 @@
     "plugin_file_service_error_101": "一つのファイルだけにしてください！",
     "plugin_file_service_error_102": "ファイルが大きすぎます！最大容量は{{limit}}です。",
     "plugin_file_service_on_change_file": "ファイルの変更",
+    "plugin_file_service_arweave": "節約 Arweave",
```

Comment:
Please don't add machine translations, if you don't know how to translate, it's OK to leave it an empty string and runtime won't be breaking

Reply:
@Jack-Works Thank you for letting me know.